File: themes/OrchidMaterialize/src/main/java/com/eden/orchid/materialize/MaterializeModule.java
package com.eden.orchid.materialize;

import com.eden.orchid.api.registration.OrchidModule;
import com.eden.orchid.api.theme.Theme;

public class MaterializeModule extends OrchidModule {

    @Override
    protected void configure() {
        addToSet(Theme.class, MaterializeTheme.class);
    }
}

File: themes/OrchidMaterialize/src/main/kotlin/com/eden/orchid/materialize/MaterializeModule.kt
package com.eden.orchid.materialize

import com.eden.orchid.api.registration.OrchidModule
import com.eden.orchid.api.theme.Theme

class MaterializeModule : OrchidModule() {

    override fun configure() {
        addToSet(Theme::class.java, MaterializeTheme::class.java)
    }
}
